### PR TITLE
NT-8: javascript to jQuery refactoring

### PR DESCRIPTION
**Pull request by TorgashSad**
📝 **Task:** [NT-8](https://trello.com/c/BWnxB4qQ/15-nt-8-front-mainjs-javascript-to-jquery)

# Description

Changed javascript to jQuery and ajax methods

### CODE REVIEW

File: src/main/resources/static/js/main.js [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
-    }
-    xhttp.send(JSON.stringify(postObj));
+    $.post({
+        url: 'api/authorization',
+        headers: contentTypeHeader,
+        dataType: 'json',
+        data: JSON.stringify({name: playerName}),
+        success: function () {
+            renderMainWindow()
+        },
```
Comment (iamsaywhat) [inline src/main/resources/static/js/main.js:25]: Can't we do the following? This function wrapper looks like it doesn't make any sense
```suggestion
        success: renderMainWindow()
```

**Review by Steellemm — APPROVED** [review]

File: src/main/resources/static/js/main.js (hunk 1 above)
Comment (TorgashSad) [reply src/main/resources/static/js/main.js:25]: If you put it this way, "Sign in" button only works on the second click. I don't why. BTW static analyzer in IntelliJ IDEA highlights your variant.
image

Comment (iamsaywhat) [reply src/main/resources/static/js/main.js:25]: Oh, well, now I see it, `success` must be some kind of callback declaration, not the call itself.

**Review by iamsaywhat — APPROVED** [review]
GJ!

**Pull request merged**